asam_cmp_capture_module/src/stream_fb.cpp

```diff
@@ -38,6 +38,7 @@ StreamFb::StreamFb(const ContextPtr& ctx,
     , allowJumboFrames(internalInit.allowJumboFrames)
     , encoders(internalInit.encoderBank)
     , parentInterfaceUpdater(internalInit.parentInterfaceUpdater)
+    , isConfigured(false)
```

Comment:
Member initialization should be reordered in the same way it is defined in the class or I suppose using default member initialization would be even better